src/config.ts

```diff
@@ -224,6 +226,10 @@ export function normalizeConfig(spec: ConfigSpec = {}, defaultRoot?: string, wat
       : spec.style !== undefined
       ? {path: String(spec.style)}
       : {theme: normalizeTheme(spec.theme === undefined ? "default" : spec.theme)};
+  const globalStylesheets =
+    spec.globalStylesheets === undefined
+      ? defaultGlobalStylesheets()
+      : Array.from(spec.globalStylesheets as any, String);
```

Comment:
```suggestion
  const globalStylesheets =
    spec.globalStylesheets === undefined
      ? defaultGlobalStylesheets()
      : !spec.globalStylesheets
      ? []
      : Array.from(spec.globalStylesheets as any, String);
```

Accept null and false rather than crashing? Or maybe we want to ignore the option in that case, rather than support falsy values?

Reply:
Yeah my thinking was that null isn’t a valid specification, so why allow it? We don’t need to allow everything. But in that case perhaps we should throw a nice error instead of crashing arbitrarily.

Reply:
I've added a nicer error.

Reply:
I found two more places where we call Array.from unsafely, too (pages and scripts).